src/main/java/org/quantumbadger/redreader/activities/OptionsMenuUtility.java

```diff
@@ -602,6 +779,25 @@ public boolean onMenuItemClick(MenuItem menuItem) {
 		});
 	}
 
+	public static int handleShowAsActionIfRoom(final int showAsAction) {
+
+		if(showAsAction == MenuItem.SHOW_AS_ACTION_IF_ROOM) return MenuItem.SHOW_AS_ACTION_ALWAYS;
+
+		return showAsAction;
+	}
+
+	// Avoids IDE warnings about null pointers
```

Comment:
A common way to do this is with `Objects.requireNonNull()`, but this is fine :)

Reply:
Thanks for the tip. :)